### ISSUE TITLE
Style/Alias: prefer_alias has no effect

### ISSUE THREAD

**Issue by pjscopeland** [rubocop-hq/rubocop#7839]
Ruby 2.7 doesn't seem to have `alias_method` any more.

```
NoMethodError:
  undefined method `alias_method' for main:Object
```

So I changed it to `alias`, but Rubocop changed it back.

So I went and found the rules for the `Style/Alias` cop. Turned out it wasn't present in my config file, it was using the default: `EnforcedStyle: prefer_alias` – so it shouldn't have changed it at all. Even trying to enforce that by _putting_ it in my config file, Rubocop still changes it back. I even tried using `prefer_alias_method` in case they were the wrong way round, but it _still_ changed it back.

--------

## Expected behavior

Expected `alias_method` calls to change to `alias` when `Style/Alias:EnforcedStyle` is `prefer_alias`.

## Actual behavior

Rubocop changes `alias` calls to `alias_method`, whichever `EnforcedStyle` is configured.

## Steps to reproduce the problem

Include this in `.rubocop.yml` (or, in my case, an inherited file):

```yml
Style/Alias:
  EnforcedStyle: prefer_alias
```

My code: 

```ruby
RSpec.configure do |config|
  if $rspec_version <= Gem::Version.new('2.14.8')
    alias focus_me ppc
    alias unfocus_me ppc
  else
    # ... truncated
  end
end
```

Run `rubocop --auto-correct my_file.rb`

## RuboCop version

```
0.81.0 (using Parser 2.7.0.5, running on ruby 2.7.0 x86_64-darwin17)
```

**Comment by Drenmi** [comment]
I don't think Ruby has ever had `#alias_method` outside of a class or module. 🤔 

And in  classes and modules it seems like it's still there in 2.7.0:

```ruby
RUBY_VERSION
#=> "2.7.0"

class Foo
  def bar; "hello world"; end
  
  alias_method :baz, :bar
end

foo = Foo.new

foo.bar
#=> "hello world"

foo.baz
#=> "hello world"
```
